### PR TITLE
Add note about range elements for days

### PR DESCRIPTION
**Pull request by nbialostosky**
# Description
This PR adds a disclaimer that setting up a range of days written out is not supported. 

# Reasons
Crontab says this is valid:

https://crontab.guru/#0_5_*_*_Tue-Sat

And while that might be the case for some systems, it isn't valid within a CircleCI config. Instead, the values need to be comma separated digits like: https://crontab.guru/#0_5_*_*_2,3,4,5

### CODE REVIEW

File: jekyll/_cci2/workflows.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ A valid `schedule` requires a `cron` key and a `filters` key.
 The value of the `cron` key must be a [valid crontab entry](https://crontab.guru/).
 
 **Note:**
-Cron step syntax (for example, `*/1`, `*/20`) is **not** supported. Range elements within comma-separated lists of elements are also **not** supported. 
+Cron step syntax (for example, `*/1`, `*/20`) is **not** supported. Range elements within comma-separated lists of elements are also **not** supported. In addition, range elements for days (for example, `Tue-Sat`) is **not** supported, use comma seperated digits instead.
```
Comment (thomasr8) [inline jekyll/_cci2/workflows.md:279]:
```suggestion
Cron step syntax (for example, `*/1`, `*/20`) is **not** supported. Range elements within comma-separated lists of elements are also **not** supported. In addition, range elements for days (for example, `Tue-Sat`) is **not** supported. Use comma-separated digits instead.
```

Comment (thomasr8) [reply jekyll/_cci2/workflows.md:279]: Just a couple of minor changes needed. Overall looks good.

**Comment by teesloane** [comment]
Just to cross-check, a [user submitted a PR](https://github.com/circleci/circleci-docs/pull/5000) that they are able to use the cron syntax we say is unavailable - is it the case that we don't officially support it but it still works?

**Comment by nbialostosky** [comment]
@teesloane I think we just need to improve the language around that maybe? What the docs are trying to communicate is that you can't use a range element (like `3-5`) within a comma-separated list, i.e. this doesn't work:

```
    triggers:
      - schedule:
          cron: "5 4 * * 1,3-5,6"
```

So you would need to list them all out, i.e.:

```
    triggers:
      - schedule:
          cron: "5 4 * * 1,3,4,5,6"
```

**Comment by teesloane** [comment]
Ah, got it. Thanks @nbialostosky ! I will close that user's PR and update the wording after we merge this!

**Review by thomasr8 — APPROVED** [review]

**Pull request merged**